src/Microsoft.ApplicationInsights.AspNetCore/SdkVersionUtils.cs

```diff
@@ -5,20 +5,17 @@
 
     internal class SdkVersionUtils
     {
-        internal const string VersionPrefix = "aspnetCore-";
+        internal const string VersionPrefix = "aspnet5";
```

Comment:
We can consider defining two consts under #if rather that introducing a separate method and do concatenation.